js/ui/html_editor/modules/tableResizing.js

```diff
@@ -0,0 +1,298 @@
+            this._nextLineSize = parseInt($($determinantElements[index + 1])[directionInfo.getSizeFunction]());
+        }
+    }
+
+    _dragMoveHandler(event, { $determinantElements, index, frame, direction }) {
+        const directionInfo = this._getDirectionInfo(direction);
+        const currentLineNewSize = this._startLineSize + event.offset[directionInfo.positionCoordinateName];
+
+        if(direction === 'horizontal') {
+            const nextColumnNewSize = this._nextLineSize && this._nextLineSize - event.offset[directionInfo.positionCoordinateName];
+            const isCurrentColumnHasEnoughPlace = currentLineNewSize >= this._minColumnWidth;
+            const isNextColumnHasEnoughPlace = !this._nextLineSize || nextColumnNewSize >= this._minColumnWidth;
+
+            if(isCurrentColumnHasEnoughPlace && isNextColumnHasEnoughPlace) {
+                $determinantElements.eq(index).attr(directionInfo.positionStyleProperty, currentLineNewSize + 'px');
+
+                if(this._nextLineSize) {
+                    $determinantElements.eq(index + 1).attr(directionInfo.positionStyleProperty, nextColumnNewSize + 'px');
+                }
+            }
+        } else {
+            const newHeight = Math.max(currentLineNewSize, this._minRowHeight);
+            $determinantElements.eq(index).attr(directionInfo.positionStyleProperty, newHeight + 'px');
+        }
+
+        this._updateFramePosition(frame.$table, frame.$frame);
+    }
+
+    _canChangeTableSizes(options) {
+        return options.direction === 'vertical' || this._isLastColumnResizing(options);
+    }
+
+    _isLastColumnResizing({ $determinantElements, index }) {
+        return !isDefined($determinantElements[index + 1]);
+    }
+
+    _createDraggableElement(options) {
+        this._currentDraggableElement = this.editorInstance._createComponent(options.lineSeparator, Draggable, {
+            contentTemplate: null,
+            boundary: this._canChangeTableSizes(options) ? this.editorInstance._getQuillContainer() : options.frame.$table,
+            allowMoveByClick: false,
+            dragDirection: options.direction,
+            onDragMove: ({ component, event }) => {
+                this._dragMoveHandler(event, options);
+            },
+            onDragStart: () => {
+                this._dragStartHandler(options);
+            },
+            onDragEnd: () => {
+                this._updateFramesPositions();
+                this._updateFramesSeparators();
+            }
+        });
+    }
+
+    _fixColumnsWidth(frame) {
+        const determinantElements = this._getTableDeterminantElements(frame.$table);
+
+        each(determinantElements, (index, element) => {
+            const columnWidth = $(element).outerWidth();
+            $(element).attr('width', columnWidth >= DEFAULT_MIN_COLUMN_WIDTH ? columnWidth : DEFAULT_MIN_COLUMN_WIDTH);
+        });
+
+        frame.$table.css('width', 'auto');
+    }
+
+    clean() {
```

Comment:
At the moment you should use the `addCleanCallback` method
E.g. https://github.com/DevExpress/DevExtreme/blob/0f62fd79e40504ffd9147201f5e3f8f114227863/js/ui/html_editor/modules/toolbar.js#L106